src/routes/index.ts

```diff
@@ -1,9 +1,47 @@
 import { Router } from "express";
+import { db } from "../structs/database";
+import { minioConfig as getMinioConfig } from "../structs/config";
+import * as http from "http";
 
 const router = Router();
+const minioConfig = getMinioConfig();
 
 router.get("/", (req, res) => {
   res.end("Try /recipe");
 });
 
+router.get("/health", (req, res) => {
+  // Send back sattus code 200, the API is live.
+  res.status(200);
+
+  let minioHealthRequestOptions = {
+    method: "GET",
+    host: minioConfig.endPoint,
+    port: minioConfig.port,
+    path: "/minio/health/live",
+    timeout: 5000
+  };
+
+  // Check the minio server for health
+  const minioHealthRequest = http.request(minioHealthRequestOptions, (response) => {
+    res.send({
+      mongodb: db.readyState === 1,
+      minio: response.statusCode === 200
+    });
+    res.end();
+  });
+
+  minioHealthRequest.on("timeout", () => {
+    res.send({
```

Comment:
Use `res.json` instead of `res.send`. That sets the `Content-Type` header to `application/json` instead of `text/html`

Reply:
Indeed the `res.json` always tries to send a `application/json` formatted response, but res.send does the same when a JSON object is passed instead of a string.

Reply:
Fixed in 74df2b3